src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java

```diff
@@ -14,9 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.app.impl.config;
-
-import com.google.cloud.tools.app.api.Configuration;
+package com.google.cloud.tools.app.api;
```

Comment:
Not on my account. I'm fine with you changing it now.

On Mon, Jun 6, 2016 at 5:12 PM, meltsufin <notifications@github.com> wrote:

> In src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/105#discussion_r65970028>
> :
>
> > @@ -14,9 +14,7 @@
> >   * limitations under the License.
> >   */
> >
> > -package com.google.cloud.tools.app.impl.config;
> > -
> > -import com.google.cloud.tools.app.api.Configuration;
> > +package com.google.cloud.tools.app.api;
>
> Should we then hold off renaming the app package as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/105/files/5723a07a6bb84c29656eee4b39a1bb8a36e1b524#r65970028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5Ha3BtQsiWg7szpMX0e8DRq-Ewqx6ks5qJI0sgaJpZM4IvDJS>
> .
>